django_mongodb/base.py

```diff
@@ -119,6 +119,15 @@ def _isnull_operator(a, b):
         "iregex": lambda a, b: regex_match(a, b, insensitive=True),
     }
 
+    mongo_operators_idx = {
```

Comment:
If this is all the operators we have, probably we don't need such a complex approach and can simply use a dict with pairs like` {"exact": "$eq"}` similar to `MONGO_OPERATORS` in `functions.py`.

Reply:
Yes, they are only a few much and fewer in mongo5, I will move this to indexes.py